test/integration/components/RecapComponent.js

```diff
@@ -0,0 +1,8 @@
+firstPersonBirthdate: 'individu-block:nth-of-type(1) strong',
+secondPersonBirthdate: 'individu-block:nth-of-type(2) strong',
+thirdPersonBirthdate: 'individu-block:nth-of-type(3) strong',
+fourthPersonBirthdate: 'individu-block:nth-of-type(4) strong',
```

Comment:
The best you can do is to declare a helper function in a `Fixture` file and call it:

```js
// IndividuFixture
function nthIndividu(index) {
     return 'individu-block:nth-of-type(' + index + ') strong';
}

// RecapComponent
firstPersonBirthdate: nthIndividu(1),
secondPersonBirthdate: nthIndividu(2),
…
```